src/TextField/TextField.spec.js

```diff
@@ -119,9 +114,9 @@ describe('<TextField />', () => {
     });
   });
 
-  describe('prop: inputProps', () => {
+  describe('prop: InputProps', () => {
     it('should apply additional properties to the Input component', () => {
-      const wrapper = mount(<TextField inputProps={{ readOnly: true }} />);
+      const wrapper = mount(<TextField InputProps={{ readOnly: true }} />);
```

Comment:
Should this be `<TextField InputProps={{ { input: readOnly: true } }} />`?

Reply:
This is one of those that could actually be either.  Since `Input` props explicitly lists it and propagates it to `input`, it works both ways.  Thinking of it as a fully-linked hierarchy, I can see why you go to `inputProps`.